scripts/produce-sha256

```diff
@@ -16,14 +16,16 @@ for f in packages/*; do
     find ${f} -type f | xargs sha256sum > ./sha256sum-new/$(basename -- ${f})/$(basename -- ${f}).sum
 done
 
+# If there were no sha256sums in the first place, output the name of every chart
 if [ ! -d sha256sum ]; then
     cp -R sha256sum-new sha256sum
     for f in packages/*; do
         echo $(basename -- ${f})
     done
 fi
 
-for f in sha256sum/*; do 
+# For all the new sha256sums that were generated, compare them and see if the existing assets need to be rebuilt
+for f in sha256sum-new/*; do
     if ! cmp -s ./sha256sum-new/$(basename -- ${f})/$(basename -- ${f}).sum ${f}/$(basename -- ${f}).sum; then
```

Comment:
I think you have to swap this too, you are comparing same file here

Reply:
good catch!